README.md

```diff
@@ -314,6 +314,12 @@ Options: `session`: Instance of a Rails session
 
 For applications that want to keep access tokens in the user's session.
 
+:warning:
```

Comment:
I'd also add a note that the Rails session store could in fact be a cookie which would lead to tokens being stored client side, possibly unencrypted depending on settings.